index.html

```diff
@@ -1275,7 +1275,70 @@ <h2>The <dfn>InteractionOutput</dfn> interface</h2>
       </ol>
     </section>
   </section>
-  <section> <h3>Using {{InteractionInput}} and {{InteractionOutput}}</h3>
+  
+  <section data-dfn-for="ActionInteractionOutput">
+    <h2>The <dfn>ActionInteractionOutput</dfn> interface</h2>
+    <p>
+      Belongs to the <a>WoT Consumer</a> conformance class.
+      An {{ActionInteractionOutput}} object is always created by a consumer implementation
+      and exposes functionality to interact with long running (asynchronous) actions.<br />
+      Note: a synchronous action MAY not offer any <emph>additional</emph> sensible interaction beyond the ones from {{InteractionOutput}}. 
```

Comment:
- This `**MAY** not offer` is likely to be confused with `**MAY NOT** offer`, despite formatting, casing, etc. 
- Further, I'm not sure what `sensible` is contributing here. 
- I'm also left wondering whether there should be some indication of what other `interactions` (`sensible` or not) *might* be offered (if the `action` does offer more than those in `{{InteractionOutput}}`)?

Perhaps this rephrasing is acceptable, for the first two points above? Something additional might be needed for the last.
```suggestion
      Note: A synchronous action MAY offer only the interactions found in {{InteractionOutput}} and no others.
```

Reply:
I've tried to cover the third aspect that you've mentioned by rephrasing the sentence as follows – hope that it is a bit clearer than it was before:

```suggestion
      Note: The output of a synchronous action MAY be limited to the functionality of a regular {{InteractionOutput}} object, e.g., invoking the `cancel()` method might have an effect.
```

Reply:
I'm not against it, if its accurate, but this revision (which would cover all of my bullets) seems to have a very different meaning than the original.